doc/content/verification_and_validation/val-2b.md

```diff
@@ -4,54 +4,72 @@
 
 ## Test Description
 
-This validation problem is taken from [!cite](macaulay1991deuterium). He and co-workers conducted thermal absorption and desorption experiments, as well as implantation experiments, on wafers of polished beryllium. Of the several data sets presented, the one modeled here is that represented in Figure 12(a) in their publication. The beryllium sample was 0.4 mm thick and had an area of 104 mm$^2$. It was polished to a mirror finish and then exposed to 13.3 kPa of deuterium at 773 K for 50 hours. It was quickly cooled under a vacuum of about 1 $\mu$Pa. The cooling time constant for the apparatus is taken as 45 minutes. After removing the sample from the charging furnace, it was transferred in the air to a thermal desorption furnace where the temperature was increased from ambient (300 K) to 1073 K at the rate of 3 K/min. This was done under vacuum, and the pressure of the chamber was monitored by the residual gas analysis and calibrated against standard leaks. In that way, the emission rate from the sample could be measured as a function of temperature. Data from that measurement, given in Figure 12 (a) of their publication are reproduced in [val-2b_comparison] here.
+This validation problem is taken from [!cite](macaulay1991deuterium) and is part of the validation suite of TMAP4 and TMAP7 [!citep](longhurst1992verification,ambrosek2008verification). We here reproduce, with some updates, the case modeled in [!cite](longhurst1992verification,ambrosek2008verification).
 
+R.G. Macaulay-Newcombe et al. conducted thermal absorption and desorption experiments, as well as implantation experiments, on wafers of polished beryllium.
+Of the several data sets presented, the one modeled here is run 2a1 represented in Figure 2(a) in their publication. The beryllium sample was 0.4 mm thick and had an area of 104 mm$^2$, as illustrated in [val-2b_schematic].
+It was polished to a mirror finish and then exposed to 13.3 kPa of deuterium at 773 K for 50 hours. It was quickly cooled under a vacuum of about 1 $\mu$Pa. The cooling time constant for the apparatus is taken as 45 minutes. After removing the sample from the charging furnace, it was transferred in the air to a thermal desorption furnace where the temperature was increased from ambient (300 K) to 1073 K at the rate of 3 K/min. This was done under vacuum, and the pressure of the chamber was monitored by the residual gas analysis and calibrated against standard leaks. In that way, the emission rate from the sample could be measured as a function of temperature. The sample pressure and temperature histories are shown in [val-2b_temperature_pressure_history].
```

Comment:
> Where do you see the cooling time constant in the Macaulay-Newcombe paper? The discussion of the parameters for this sample in their paper (page 2, paragraphs 3 and 4) doesn't mention it, and neither does the discussion in Section 3.2.

It is not specified. This is an assumption first made in the TMAP4 case. I'll specify this here. 

> The cooling period is also just short enough to get to ambient temperature, but is much shorter than the timeframe considered in the paper (>24h between removal from the charging furnace to the ramp up in the desorption furnace). How do you square this?

Alright, great question - I read the original paper 50 times and tried to make sense of what was described, but I'm still quite unsure about the experimental procedure. I couldn't figure out if the baking, cooldowns, and rest procedures were applied to the sample itself or the "empty" chamber to clean up any impurities and prepare it for the experiment. In parallel, TMAP4/TMAP7 used 40 minutes, which was not enough time to decrease the temperature to the initial temperature of the desorption phase (300 K). To me, it wouldn't make sense to start the head up at 300 K if the sample itself was not yet at 300 K, so I picked a time that was long enough to bring the temperature of the sample to around 300K, but did not increase the length of the history unnecessarily (nothing significant happens at 300 K because the kinetics are very slow). 

I can be more transparent about the uncertainty in the history in the documentation.

Reply:
Done